orchestra/static/orchestra/common/components/modules.js

```diff
@@ -0,0 +1,11 @@
+(function () {
```

Comment:
The idea was to nicely collect all the reusable task view components (e.g., rich text editor, checklists) into a single directory. I nested it inside `common` because I can imagine there would be other unrelated common directives we'd want to include.

Maybe `components` should live inside the `task` static files instead, since they'll probably only be used in custom task views? Though you could imagine using a rich text editor in the admin or something...